lua/nvim-lsp-installer/path.lua

```diff
@@ -0,0 +1,38 @@
+local uv = vim.loop
+
+local sep = (function()
```

Comment:
are you sure no helper already exists? otherwise, couldn't you use `os.platform` and for win32 use `\` otherwise `/` ?

Reply:
Pretty sure. This was copied from plenary.nvim actually.

>  couldn't you use os.platform and for win32 use \ otherwise / ?

Isn't that pretty much exactly what it's doing right now though?